surfer/viz.py

```diff
@@ -356,6 +361,18 @@ def __init__(self, subject_id, hemi, surf, curv=True, title=None,
             width = config_opts.get("width", size)
             height = config_opts.get("height", size)
             size = (width, height)
+        # Keep backwards compatability
+        if curv is not None:
+            msg = ("The `curv` keyword has been deprecated and will "
+                   "be removed in future versions. You should update your "
+                   "code and use the `cortex` keyword to specify how the "
+                   "brain surface is rendered. Setting `cortex` to `None` "
+                   "will reproduce the previous behavior when `curv` was "
+                   "set to `False`. To emulate the previous behavior for "
+                   "cases where `curv` was set to `True`, simply omit it.")
+            warn(msg)
```

Comment:
I've added the category DeprecationWarning to warnings for the use of config_opts and curv. Note that the default behavior for DeprecationWarning is that it's ignored, which makes these warnings somewhat less visible now.